template.conf

```diff
@@ -76,11 +76,11 @@
         {% if site.autoindex %}
           autoindex on;
           {% if site.backends.length %}
-            try_files $uri $uri/ @proxy-{{ site.shortname }}-{{ options.port }};
+            try_files $uri $uri/ @proxy-{{ site.shortname }}-{{ options.port }}{% if options.permanent %} permanent{% endif %};
           {% endif %}
         {% else %}
           {% if site.backends.length %}
-            try_files $uri @proxy-{{ site.shortname }}-{{ options.port }};
+            try_files $uri @proxy-{{ site.shortname }}-{{ options.port }}{% if options.permanent %} permanent{% endif %};
```

Comment:
This is a try_files directive, not a redirect. I don't think permanent would work here. I think you meant to do this elsewhere, where the various redirect options are handled. Also this needs to be tested on an actual server.

Reply:
https://www.nginx.com/blog/creating-nginx-rewrite-rules/#Examples&nbsp;%E2%80%93-Standardizing-the-Domain-Name

Reply:
It's not the recommended way to solve but it is a way. 🤷🏻‍♂️

Reply:
You're right! I meant to put this at the end of a `rewrite`. I shall fix.